apps/client/app/purchase-orders/page.tsx

```diff
@@ -1,8 +1,112 @@
+  order_date: string;
+  expected_delivery_date: string;
+  created_at: string;
+  updated_at: string;
+  purchase_order_line_items: PurchaseOrderLineItem[];
+}
+
+async function getPurchaseOrders(): Promise<PurchaseOrder[]> {
+  const res = await fetch('http://localhost:3100/api/purchase-orders', {
+    cache: 'no-cache',
+  });
+  if (!res.ok) {
+    throw new Error('Failed to fetch data');
+  }
+
+  const body = res.json();
+  return body;
+}
 
 export default async function Index() {
+  const purchaseOrders = await getPurchaseOrders();
+
+  const sortedPurchaseOrders = sort(purchaseOrders).asc(
+    (po) => po.expected_delivery_date
+  );
+
   return (
     <>
-      <h1 className="text-2xl">Purchase Orders</h1>
+      <div>
+        <h1 className="text-2xl m-5">Purchase Orders</h1>
+      </div>
+      <table className="table">
+        <thead className="bg-slate-200">
+          <tr>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Purchase Order Id
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Vendor
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Order Date
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Expected Delivery Date
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Total Item Quantity
+            </th>
+            <th className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400 text-left">
+              Total Price
+            </th>
+          </tr>
+        </thead>
+        <tbody className="bg-white dark:bg-slate-800">
+          {sortedPurchaseOrders.map((purchaseOrder: PurchaseOrder) => (
+            <tr key={purchaseOrder.id}>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                <Link
+                  href={'/purchase-orders/' + purchaseOrder.id}
+                  className="underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
+                >
+                  {purchaseOrder.id}
+                </Link>
+              </td>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                {purchaseOrder.vendor_name}
+              </td>
+              <td className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400">
+                {moment(purchaseOrder.order_date).format('YYYY-MM-DD')}
```

Comment:
Did you consider creating a `<Date value={purchaseOrder.order_date} />` component similar to the `<Currency />` component to reduce repetition?